apps/src/code-studio/clientState.js

```diff
@@ -37,9 +38,13 @@ clientState.reset = function() {
   } catch (e) {}
 };
 
+/**
+ * Clear progress-related values from session storage.
+ */
 clientState.clearProgress = function() {
   sessionStorage.removeItem('progress');
   sessionStorage.removeItem('lines');
+  removeItemsWithPrefix(sessionStorage, 'source_');
```

Comment:
can you comment on this change? explaining this line in the header comment might be nice too. the first two lines are pretty clear, but this one is not self-explanatory

Reply:
never mind. the `sourceForLevel` function below was collapsed, but seeing it there for context make this more self-explanatory